doc/api/esm.md

```diff
@@ -54,6 +54,14 @@ property:
 
 ## Notable differences between `import` and `require`
 
+### Mandatory file extensions
+
+You must provide a file extension when using the `import` keyword.
```

Comment:
We usually avoid `you` in API docs, as per [style guide](https://github.com/nodejs/node/blob/master/doc/STYLE_GUIDE.md).